src/Microsoft.ML.Transforms/MissingValueHandlingTransformer.cs

```diff
@@ -56,7 +56,7 @@ public enum ReplacementKind : byte
             Max = Maximum,
         }
 
-        public sealed class Arguments : TransformInputBase
+        public sealed class Options : TransformInputBase
```

Comment:
>public [](start = 8, length = 6)

internal

Reply:
I made the entire class internal so this is not an issue.

---
In reply to: [253752265](https://github.com/dotnet/machinelearning/pull/2363#discussion_r253752265) [](ancestors = 253752265)